build.gradle

```diff
@@ -22,7 +22,7 @@ ext {
     junit4Version = '4.12'
     flexmarkVersion = '0.32.20'
     jettyServerVersion = '9.2.24.v20180105'
-    orientDbVersion = '2.2.33'
+    orientDbVersion = '2.2.35'
```

Comment:
Let's go 2.2.36
https://github.com/orientechnologies/orientdb/wiki/OrientDB-2.2-Release-Notes#2236---june-12-2018

Reply:
See #471

Reply:
https://github.com/jbake-org/jbake/pull/471/files#diff-c197962302397baf3a4cc36463dce5eaR25

Reply:
Well maybe it could get in in a separate PR? Reviewing an upgrade is way easier than a whole PR :)

Reply:
jup. I'll take it out of the pr then. it's not necessary for the performance optimization anyways.